fluent-langneg/src/headers.js

```diff
@@ -0,0 +1,8 @@
+
+export default function parseAcceptedLanguagesHeaderIntoArray(string = '') {
+  if (typeof string !== 'string') throw new TypeError('Argument must be a string');
+  const tokens = string.split(',').map(t => t.trim());
+  return tokens.filter(t => t !== '').map(t => {
+    return t.split(';')[0];
```

Comment:
Do you plan to extend this in the future? The RFC says the q-values are in fact important.